doc/Generalmammalianlifehistoryliteraturesummary.Rmd

```diff
@@ -0,0 +1,46 @@
+---
+title: "General Mammalian Life History Literature Summary"
+output: html_notebook
+---
+For this part in the introduction I stated broad and focused on the costs of reproduction in mammals. I started off my saying there is cost and then I went more into the fast-slow continuum theory and how mammals are found all throughout the spectrum and I found two article that listed one species at one end of the spectrum and another on the other end of the spectrum. I feel like we can build off this "cost of reproduction seen in mammals"" and the fact that mammals differ on the continuum to lead into parent-offspring conflicts. 
+
+# Trade-Offs in Life History Evolution (Stearns, 1989)
```

Comment:
Don't forget to add an empty line between the heading and the following paragraph